targets/TARGET_STM/README.md

```diff
@@ -1,14 +1,57 @@
-## README for mbed-os STM32 targets
+# README for mbed-os STM32 targets
 
-### MBED Wiki pages
+## ST TOOLS
 
-https://os.mbed.com/teams/ST/wiki/
+### USB drivers
+
+Mandatory: get the latest USB driver in order to make available all the USB interfaces provided by the ST-LINK:
+- ST Debug
+- Virtual COM port
+- ST Bridge interfaces
+
+https://www.st.com/en/development-tools/stsw-link009.html
+
+### ST-Link FW
+
+Mandatory: get the latest ST-Link Firmware:
+
+https://www.st.com/en/development-tools/stsw-link007.html
+
+Note that with the latest FW version, you are able to check the version number easily with a simple "mbedls":
+
+```
+$ mbedls
+| platform_name       | platform_name_unique   | mount_point | serial_port | target_id                | interface_version |
+|---------------------|------------------------|-------------|-------------|--------------------------|-------------------|
+| DISCO_H747I         | DISCO_H747I[0]         | D:          | COM13       | 081402210D03E72132477E08 | V3J7M2            |
+| DISCO_L475VG_IOT01A | DISCO_L475VG_IOT01A[0] | E:          | COM9        | 07640221683B630A577FF553 | V2J37M26          |
+```
```

Comment:
Need this merged (https://github.com/ARMmbed/mbed-os-tools/pull/230) but not a blocker